dist.Dockerfile

```diff
@@ -8,6 +8,10 @@ ARG gid=10001
 COPY . /var/cartography
 WORKDIR /var/cartography
 
+RUN apt-get update --fix-missing
+RUN apt --allow-unauthenticated  update -y
```

Comment:
I can't think of a good reason why `--allow-unauthenticated` would be needed here.